### PR TITLE
 Fixed multiple bugs and improvement

### PR DESCRIPTION
**Pull request by Guillaume28**
#53 #51 #49 #47 #35 #28 #26 #19 #14 #3

Many issues with little bugs and several improvement.
Select2 implemented also.

### CODE REVIEW

File: service-monitoring/configs.xml [hunk 1]
```diff
@@ -4,15 +4,18 @@
   <email>contact@centreon.com</email>
   <website>http://www.centreon.com</website>
   <description>Widget for displaying service monitoring information</description>
-  <version>1.4.2</version>
+  <version>1.4.9</version>
```
Comment (kduret) [inline service-monitoring/configs.xml]: Do not increment version for each fix

File: service-monitoring/data.js [hunk 2]
```diff
@@ -10,9 +10,9 @@ function loadPage()
     jQuery.ajax("./src/index.php?widgetId=" + widgetId + "&page=" + pageNumber, {
         success: function (htmlData) {
             jQuery("#serviceMonitoringTable").empty().append(htmlData);
-            var hostMonitoringTable = jQuery("#serviceMonitoringTable").find("img, style, script, link").load(function () {
+            var serviceMonitoringTable = jQuery("#serviceMonitoringTable").find("img, style, script, link").load(function () {
                 var h = document.getElementById("serviceMonitoringTable").scrollHeight + 50;
-                parent.iResize(window.name, h);
+                ResizeFrame(window.name, h);
```
Comment (kduret) [inline service-monitoring/data.js]: why rename method ?

File: service-monitoring/src/toolbar.php [hunk 3]
```diff
@@ -1,154 +1 @@
...
                                });
            } else {
                alert("<?php echo _('Please select one or more items'); ?>"); 
                return false;
            }
            $(".toolbar").val(0);
        }
    });
});
</script>
```
Comment (kduret) [inline service-monitoring/src/toolbar.php]: Bad encode

Comment (Guillaume28) [reply service-monitoring/src/toolbar.php]: Encode is good now

**Pull request closed without merging**